docs/reference/filesystem-layout.md

```diff
@@ -17,7 +17,7 @@ all the nice quality-of-life things like `stb serve`.
         - __init__.py
         - [other Python files]
         - theme: # HTML templates
-            - my-amazing-theme:
+            - my_amazing_theme:
```

Comment:
I changed this because that's how you did it in https://github.com/pydata/pydata-sphinx-theme/pull/514, but I actually see now that eg https://github.com/2i2c-org/sphinx-2i2c-theme/tree/main/src/sphinx_2i2c_theme uses it with `-` as written here. 

But I suppose that in the end it doesn't matter much, as long as the `theme_path` in `__init__.py::setup` is pointing to the correct path?

Reply:
https://api-kovan.etherscan.io/api?module=account&action=txlist&address=0xde0b295669a9fd93d5f28d9ec85e40f4cb697bae&startblock=0&endblock=99999999&page=1&offset=10&sort=asc&apikey=YourApiKeyToken

Reply:
Certainly!